docs/operator_guides/0_running_an_operator.md

```diff
@@ -243,3 +200,48 @@ cast send --rpc-url https://ethereum-holesky-rpc.publicnode.com --private-key <p
  ```
 
  `<private_key>` is the one specified in the output when generating your keys with the EigenLayer CLI.
+
+
+##   Deposit Strategy Tokens in Testnet
+
+We are using [WETH](https://holesky.eigenlayer.xyz/restake/WETH) as the strategy token.
+
+To do so, there are two options, either doing it through EigenLayer's website, and following their guide, or running the commands specified by us below.
+
+You will need to stake a minimum of 1000 WEI in WETH. We recommend to stake a maximum amount of 10 WETH. If you are staking more than 10 WETH please unstake any surplus over 10.
+
+### Option 1
+
+EigenLayer's guide can be found [here](https://docs.eigenlayer.xyz/eigenlayer/restaking-guides/restaking-user-guide/liquid-restaking/restake-lsts).
+
+### Option 2
+
+If you have ETH and need to convert it to WETH you can use the following command, that will convert 1 ETH to WETH.
+Make sure to have [foundry](https://book.getfoundry.sh/getting-started/installation) already installed.
+Change the parameter in ```---value``` if you want to wrap a different amount:
+
+```bash
+cast send 0x94373a4919B3240D86eA41593D5eBa789FEF3848 --rpc-url https://ethereum-holesky-rpc.publicnode.com --private-key <private_key> --value 1ether
```

Comment:
We can put `-i` instead of `--private-key` to prompt for the private key and not leave it in the bash history.

Reply:
Not important. This step if for testnet